docs/flink-getting-started.md

```diff
@@ -747,6 +747,44 @@ FlinkSink.builderFor(
   .append();
 ```
 
+### monitoring metrics
+
+The following Flink metrics are provided by the Flink Iceberg sink.
+
+Parallel writer metrics are added under the sub group of `IcebergStreamWriter`. 
+They should have the following key-value tags.
```

Comment:
Would it be worthwhile to provide an example set of metrics values in a separate table for both objects?

Reply:
I am not sure if it is too helpful with a table for two rows of key-values.

Reply:
Do I understand correctly that the metrics key would look like this: `iceberg.my_db.my_table.0.lastFlushDurationMs`

Reply:
Not sure what `sub group` means. Maybe I just have to look it up

Reply:
https://nightlies.apache.org/flink/flink-docs-master/docs/ops/metrics/#user-scope

```Java
counter = getRuntimeContext()
  .getMetricGroup()
  .addGroup("MyMetrics")
  .counter("myCounter");

counter = getRuntimeContext()
  .getMetricGroup()
  .addGroup("MyMetricsKey", "MyMetricsValue")
  .counter("myCounter");
```